src/types/TodoUpdateContext.ts

```diff
@@ -0,0 +1,9 @@
+import { Todo } from './Todo';
+
+export type TodoUpdateContextType = {
```

Comment:
```suggestion
export interface TodoUpdateContextType  {
```
use interface prefer